cmd/solvercli/cmd/settlements.go

```diff
@@ -0,0 +1,90 @@
+package cmd
+
+import (
+	"fmt"
+	"math/big"
+
+	"github.com/skip-mev/go-fast-solver/ordersettler"
+	"github.com/skip-mev/go-fast-solver/shared/clientmanager"
+	"github.com/skip-mev/go-fast-solver/shared/keys"
+	"github.com/skip-mev/go-fast-solver/shared/lmt"
+	"github.com/skip-mev/go-fast-solver/shared/txexecutor/cosmos"
+	"github.com/spf13/cobra"
+	"go.uber.org/zap"
+)
+
+type chainPair struct {
+	source      string
+	destination string
+}
+
+type groupedSettlement struct {
+	amount *big.Int
+	count  int
+}
+
+var settlementsCmd = &cobra.Command{
+	Use:   "settlements",
+	Short: "Show pending settlement amounts across chains",
+	Run: func(cmd *cobra.Command, args []string) {
+		ctx := setupContext(cmd)
+		keysPath, err := cmd.Flags().GetString("keys")
```

Comment:
one of the desired goals that was expressed for the settlement command is that we should be able to check the pending settlements for any arbitrary solver. can we do that if we require the private keys to be specified?

Reply:
yes, this is for chandra
<img width="679" alt="Screenshot 2024-12-05 at 2 16 23 AM" src="https://github.com/user-attachments/assets/3da1c2e2-3f81-49a2-8c5f-7d5249df6f20">

Reply:
i feel like we should add an option to not use a keys file and instead use a flag to specify the osmosis address and evm address. using the keys file is nice (and probably a good default for solvers) when checking your own balances, but for us when we want to check on other solvers balances, creating keys files for all of them just to check the inventory/settlements is going to get tiring)

Reply:
im assuming you are requiring the keys file since the client manager needs it, but you could probably make a new constructor that allows a smaller set of functions (non signing functions) that doesnt require the keys to be specified

Reply:
I ran this without a keys file setup specifically for chandra, just the latest default local/keys.json that is in main. the keystore values aren't really being used here, its the addresses configured in the config.yml file

i agree that it would be cleaner to have a separate client manager, can this be done in a separate PR though? this one is quite big already and this seems a lower priority task, since it doesn't affect us checking other solvers balances

I updated the balances command though to accept custom address flags, and default to the addresses in the config otherwise. Doing this for settlements is a bigger refactor since we depend on the config directly in the ordersettler process, and would be out of scope for this PR if we want to do that. The current experience for us to check other solvers balances is just needing to update the config solver_address field, which isnt too painful imo